packages/clerk-js/src/ui/localization/__tests__/applyTokensToString.test.ts

```diff
@@ -23,6 +26,25 @@ describe('applyTokensToString', function () {
     ],
     ['', ''],
     [undefined, ''],
+    ["Last {{ date | weekday('en-US','long') }} at {{ date | timeString('en-US') }}", 'Last Friday at 10:00 PM'],
```

Comment:
Nit picking here but I know you don't mind it 🙃 
Can we split the relative days to a separate `describe` block since they are way more complex than the rest?
Also, don't forget to add some "broken" cases, eg: 
- weekDay without params
- weekDay after a non-date token
- multiple modifiers with and without params, eg: `date | weekDay(...) | titleize`
-